src/main/java/gim/ui/HelpWindow.java

```diff
@@ -15,9 +16,23 @@
  */
 public class HelpWindow extends UiPart<Stage> {
 
-    public static final String USERGUIDE_URL = "https://ay2223s1-cs2103t-t15-4.github.io/tp/";
-    public static final String HELP_MESSAGE = "Refer to the user guide: " + USERGUIDE_URL;
+    public static final String USERGUIDE_URL = "https://ay2223s1-cs2103t-t15-4.github.io/tp/UserGuide.html";
+    public static final String HELP_MESSAGE = USERGUIDE_URL;
 
+    public static final String HELP_PARA = "Hi there, welcome to GIM! Here is a list of supported commands.\n"
+            + "1) :a - Adds an exercise\n"
+            + "2) :d - Deletes an exercise\n"
+            + "3) :ls - Lists all exercise entries\n"
+            + "4) :find - Finds an exercise with a particular name\n"
```

Comment:
:find do allow users to key in more than one name, perhaps we should include it in the description?

Reply:
Necessary changes have been made, thanks